src/lib/api-client.js

```diff
@@ -113,10 +113,7 @@ export function searchItem(address) {
         address
     })
     .then(data => {
-        Dispatcher.handleAction('ITEM_FOUND', {
-            address: data.item.address,
-            status: data.item.status
-        });
+        Dispatcher.handleAction('ITEM_FOUND', data);
```

Comment:
False, see: https://github.com/TheFourFifths/consus-client/blob/view-all-items-tff35/src/store/item-store.js#L16 and https://github.com/TheFourFifths/consus-client/blob/view-all-items-tff35/src/views/components/item.jsx. No where is there an extra `item` call.